cli/pkg/commands/images/describe.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2024. Open Data Hub Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package images
+
+import (
+	"fmt"
+
+	"github.com/opendatahub-io/ai-edge/cli/pkg/commands/common"
+	. "github.com/opendatahub-io/ai-edge/cli/pkg/commands/common"
+	"github.com/opendatahub-io/ai-edge/cli/pkg/commands/flags"
+	. "github.com/opendatahub-io/ai-edge/cli/pkg/commands/flags"
+	"github.com/spf13/cobra"
```

Comment:
```suggestion
    "fmt"

	"github.com/spf13/cobra"

	"github.com/opendatahub-io/ai-edge/cli/pkg/commands/common"
	"github.com/opendatahub-io/ai-edge/cli/pkg/commands/flags"
```

That's how this import group should look like after being organized.

Reply:
done